v20.1/admin-ui-cluster-overview-page.md

```diff
@@ -3,88 +3,131 @@ title: Cluster Overview Page
 toc: true
 ---
 
-The **Cluster Overview** page of the Admin UI provides details of the cluster nodes and their liveness status, replication status, uptime, and key hardware metrics. [Enterprise users](enterprise-licensing.html) can enable and switch to the [Node Map](admin-ui-cluster-overview-page.html#node-map-enterprise) view.
+The **Cluster Overview** page of the Admin UI displays key metrics about your cluster and individual nodes. These include liveness status, replication status, uptime, and hardware usage.
 
-## Cluster Overview Panel
+If you have an [enterprise license](enterprise-licensing.html), you can enable the [Node Map](#node-map-enterprise) view for a visual representation of your cluster.
 
-<img src="{{ 'images/v20.1/admin-ui-cluster-overview-panel.png' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+## Cluster Overview panel
 
-The **Cluster Overview** panel provides the following metrics:
+Use the **Cluster Overview** panel to quickly assess the capacity and health of your cluster.
+
+<img src="{{ 'images/v20.1/admin-ui-cluster-overview-panel.png' | relative_url }}" alt="CockroachDB Admin UI cluster overview" style="border:1px solid #eee;max-width:100%" />
 
 Metric | Description
 --------|----
-Capacity Usage | <ul><li>Used capacity: The storage capacity used by CockroachDB (represented as a percentage of total storage capacity allocated across all nodes).</li><li>Usable capacity: The space available for CockroachDB data storage (i.e., the storage capacity of the machine excluding the capacity used by the Cockroach binary, operating system, and other system files). </li></ul>
-Node Status | <ul><li>The number of [live nodes](#live-nodes) in the cluster.</li><li>The number of suspect nodes in the cluster. A node is considered suspect if its liveness status is unavailable or the node is in the process of decommissioning.</li><li>The number of [dead nodes](#dead-nodes) in the cluster.</li>
+Capacity Usage | See [Capacity metrics](#capacity-metrics).<br><br><ul><li>Used: The total disk space in use by CockroachDB data across all nodes. This excludes the disk space used by the Cockroach binary, operating system, and other system files.</li><li>Usable: The total disk space usable by CockroachDB data across all nodes. This cannot exceed the store size, if one has been set using [`--store`](cockroach-start.html#store).</li></ul>
```

Comment:
"See Capacity metrics" could offer a little more context as to why one would want to click that link.

I might move the link below the list of terms in the spirit of a "for more information, see X".

For example "See Capacity metrics" could be moved below the list of terms and edited so it says something more like "For more detailed information about how the capacity numbers above are calculated, see [Capacity Metrics]()".  Or so.